config/routes.rb

```diff
@@ -1,20 +1,24 @@
 Rails.application.routes.draw do
 
-  root to: 'home#index'
-  resources :cards
-  resources :reviews, only: [:new, :create]
-  resources :decks do
-    get :set_current, on: :member
+  root to: 'guest/home#index'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.